landscape/explanation/glossary/distribution.md

```diff
@@ -0,0 +1,23 @@
+# Distribution
+
+In Landscape, a distribution refers to a specific release of the Ubuntu operating System. You can reference it with either the code name (for example jammy), the version number such as 22.04.4, or the actual name (Ubuntu) depending on your current use case.
+
+Distributions are useful for many reasons in Landscape. Many Landscape APIs require you to define a distribution in order to use them. Distributions are also important if you want to perform an action such as [repository mirroring](https://ubuntu.com/landscape/docs/explanation-about-repository-mirroring).
+
+## Landscape APIs That Require a Distribution
+The following is a list of some Landscape APIs that need a distribution to function properly:
+
+- **Repositories**: The methods under repositories allow you to manage different repositories. Some of these methods include `CreateDistribution`, and `GetDistributions`. In both cases, you reference the distribution by its name. Read [repository methods for more information](https://ubuntu.com/landscape/docs/api-repositories)
+
+- **Reporting Methods**: These methods allow you to do reporting on selections of computers. They usually take the distribution as a query parameter. For example, here's how to use the `GetCSVComplianceData` method:
+    ```shell
+    ?action=GetCSVComplianceData&query=distribution:12.04
```

Comment:
```suggestion
```

Reply:
I removed the examples for similar reasons as I mentioned above - it's just easier to maintain the document if we keep that information on those API document pages. 

You did a good job with your examples though! :smiley: